units/systemd-binfmt.path

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Path activation for additional binary formats
+Documentation=man:systemd-binfmt.service(8) man:binfmt.d(5)
+Documentation=https://www.kernel.org/doc/Documentation/binfmt_misc.txt
+
+[Path]
+DirectoryNotEmpty=/lib/binfmt.d
+DirectoryNotEmpty=/usr/lib/binfmt.d
+DirectoryNotEmpty=/usr/local/lib/binfmt.d
+DirectoryNotEmpty=/etc/binfmt.d
+DirectoryNotEmpty=/run/binfmt.d
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Oops, that's a leftover from my local testing.

In fact, since the buildsystem makes static symlinks for this unit (and the other builtin) ones, that `Install` section should probably just completely disappear?